play-json/src/main/scala/com/lightbend/lagom/scaladsl/playjson/JsonSerializationFailed.scala

```diff
@@ -4,6 +4,8 @@
 
 package com.lightbend.lagom.scaladsl.playjson
 
+import scala.collection.Seq
+
```

Comment:
wat

Reply:
This is one of the changes in 2.13.  In 2.12 Seq is not immutable.  This import reset Seq in this file to the not immutable one so it works on both 2.12 and 2.13.